docs/FIDO2-API.md

```diff
@@ -84,7 +84,7 @@ List of authenticators for provided user and application.
 }
 ```
 
-##### Response Params
+##### Description of Response Params
```

Comment:
Docucheck is not smart enough and uses only `} else if lowercasedTitle.hasPrefix("response") {`
https://github.com/wultra/docucheck/blob/develop/Sources/DocuCheckLib/Application/DatabaseOperations/Filters/BuildApiDocFilter.swift#L173